src/CommunicationApi/Services/WhatsappHandlerService.cs

```diff
@@ -74,14 +74,31 @@ public async Task<WhatsappResponse> ProcessRequest(Dictionary<string, string> pa
             WhatsappMessage whatsappMessage)
         {
             string responseMessage;
-            object[] pars = {userInfo.Name};
+            object[] pars = { userInfo.Name };
             bool accepted = false;
             switch (userCommand)
             {
+                case UserCommand.BoxStatus:
+                    _logger.LogWarning("The user {phoneNumber} asked for status of box {boxId}",
+                        userInfo.PhoneNumber, userInfo.BoxInfo?.BoxId);
+                    if (!String.IsNullOrEmpty(userInfo.BoxInfo?.BoxId))
+                    {
+                        var lastConnectedDateTime = await _boxStore.GetLastConnectedDateTime(userInfo.BoxInfo.BoxId);
+
+                        pars = new object[] { userInfo.Name, lastConnectedDateTime.ToString("d MMM yyyy HH:mm:ss") };
+                        responseMessage = "De box is voor het laatst geraadpleegd op {1}, {0}.";
+                    }
+                    else
+                    {
+                        pars = new object[] { userInfo.Name };
+                        responseMessage = "U bent nog niet gekoppeld aan een box, {0}";
+                    }
```

Comment:
Let's move all of the code from the cases to dedicated methods to make it more readible maybe?

If you prefer to park this to a dedicated PR, that's fine for me as well!

Reply:
I don't mind to do it as part of this PR, but there are multiple issues/PR's for these commands, so it will be a method at a time for each issue/PR.
And this request to move applies to both system- as usercommands? Or will the latter be moved to a bot as of #57?

Reply:
I think a bot as well but I'm not sure. Let's resolve this for now!